src/main/java/frc/robot/RobotContainer.java

```diff
@@ -196,7 +202,7 @@ private void configureBindings() {
     // Shooter commands
     if (Constants.enabledSubsystems.shooterEnabled
```

Comment:
This if/else block is probably more readable as:

```java
if (shooterEnabled && triggerEnabled) {
  if (visionEnabled) {
    ...
  } else {
    ...
  }
}
```

Reply:
Making this change would also allow you to extract `shooterSubsystem.setDefaultCommand(shooterSubsystem.shooterIdle());` to the outer if, and only write it once:

```java
if (shooterEnabled && triggerEnabled) {
  shooterSubsystem.setDefaultCommand(shooterSubsystem.shooterIdle());
  if (visionEnabled) {
    ...
  } else {
    ...
  }
}
```

Reply:
Done